Source/SwiftLintFramework/Rules/TodoRule.swift

```diff
@@ -20,4 +22,21 @@ struct TodoRule: Rule {
                 reason: "TODOs and FIXMEs should be avoided")
         }
     }
+
+    public var ruleName = "Todo Rule"
+
+    public var ruleDescription = "This rule checks whether you removed all TODOs and FIXMEs."
+
+    public var correctExamples = [
```

Comment:
I'm afraid this is where mixing tests with usage examples start to get confusing. It's not like the "correctExamples" in this case are recommended style, they're more just tests where we don't want the rule to trigger.

Maybe just changing the semantics a bit to be more like "these examples trigger the rule" and "these examples don't".